src/v/archival/archival_policy.cc

```diff
@@ -188,10 +188,10 @@ archival_policy::lookup_result archival_policy::find_segment(
         // Skip forward if we hit a gap or compacted segment
         for (auto i = set.begin(); i != set.end(); i++) {
             const auto& sg = *i;
-            if (start_offset < sg->offsets().base_offset) {
+            if (start_offset < sg->offsets().get_base_offset()) {
```

Comment:
can we move the mechanical renames into a separate commit?

Reply:
Done!